application/plugin_base/terminal.py

```diff
@@ -1,9 +1,11 @@
 import gi
 
-# require specific version of Vte
-gi.require_version('Vte', '2.90')
+from gi.repository import Gtk, Gdk, Vte
 
-from gi.repository import Gtk, Gdk, Vte, GConf
+try:
+	from gi.repository import GConf as gconf
```

Comment:
That is a pattern I saw being used elsewhere:

try:
   from gi.repository import GConf as gconf
except:
   gconf = None

and later in the code use e.g. "if gconf is None" ...

Christian

PS: I will submit a modification for lower case comments - is this a
Python thing?